bridge-adaptivity/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     container_name: nginx_BFA
     build: ./nginx
     ports:
-      - "80:80"
+      - "8080:80"
```

Comment:
This is because of our stage configuration Gamanenko's quick hack: we have system nginx which listens 80 and proxies all to 8080.